crates/forge/bin/cmd/create.rs

```diff
@@ -380,9 +380,17 @@ impl CreateArgs {
             let ty = input
                 .resolve()
                 .wrap_err_with(|| format!("Could not resolve constructor arg: input={input}"))?;
+
+            // negative ints must be escaped on CLI, so we strip '
+            let arg = if matches!(ty, DynSolType::Int(_)) {
+                arg.trim_start_matches("'").trim_end_matches("'")
+            } else {
+                arg.as_str()
+            };
```

Comment:
don't really like this, could passing through `-- -1` work?

Reply:
no calp just interprets `-<val>` as an arg not value

> error: unexpected argument '-5' found

Reply:
This should do it then on `constructor_args`: https://docs.rs/clap/latest/clap/struct.Arg.html#method.allow_hyphen_values

Reply:
cool TIL